CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.6.1 (Jul 20, 2023)
+
+IMPROVEMENTS
+* Bump Consul OSS image version to 1.15.4 and Consul enterprise version to 1.15.4-ent
+* Bump envoy image to 1.23.1
```

Comment:
This could be Envoy 1.23.10: https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.23/v1.23

We should also mention these are the default versions, e.g.

```
* Bump default Envoy image to 1.23.10
```